### PR TITLE
chore: add target framework override capabilities

### PR DESCRIPTION
**Pull request by kazo0**

### CODE REVIEW

File: src/Uno.CrossTargeting.props [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<Project ToolsVersion="15.0">
+	<!-- <Import Project="$(SolutionDir)\crosstargeting_override.props" Condition="exists('$(SolutionDir)\crosstargeting_override.props')" /> -->
```
Comment (agneszitte-nventive) [inline src/Uno.CrossTargeting.props:2]: normal that this is commented @kazo0 ?

**Review by agneszitte-nventive — APPROVED** [review]
@kazo0 can we add also target framework override capabilities for Uno.Themes as I don't think it's already done no ?

**Comment by kazo0** [comment]
@agneszitte-nventive yep that's what I'll do next  :)

**Comment by agneszitte-nventive** [comment]
> @agneszitte-nventive yep that's what I'll do next :)

Thanks a lot @kazo0 !

**Pull request merged**